registration/src/main/resources/templates/api/v1/registration.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
how do you use this template?

Reply:
for now it is just an example of JSON for particular operation

Reply:
such examples should be generated automatically in REST Docs documentation and embedded into service description

Reply:
done